src/filters/filters.less

```diff
@@ -98,3 +98,55 @@ pf-filter-panel {
     padding-left: 4px;
   }
 }
+
+.filter-pf {
+  &.inline-filter-pf {
+    flex: 1 1 100%;
+    margin: (@grid-gutter-width / 2) (@grid-gutter-width / 2) (@grid-gutter-width / 4) 0;
+
+    pf-filter-fields,
+    pf-filter-results {
+      display: inline-block;
+    }
+
+    .form-group {
+      margin-bottom: 0;
+      @media (min-width: 345px) {
```

Comment:
Is there a variable for this?  Thought this would be why you @imported bootstrap variables.less?

Reply:
Are there no patternfly variables which denote the min-widths at diff screen sizes?  If not, we should file an issue and add our own, then we don't have to worry about bootstrap variables being overwritten.

Reply:
Removing as this is unnecessary. Thanks.